tests/test_install.sh

```diff
@@ -1,7 +1,14 @@
 #/usr/bin/env sh
 set -e
 
+
 make="make -C $lz4_root"
+if [ "$unamestr" = 'Linux' ]; then
```

Comment:
where is defined `$unamestr` ?

Reply:
Ahh good catch. Accidentally removed it before pushing.